modules/osdk-deploy-olm.adoc

```diff
@@ -11,6 +11,9 @@ endif::[]
 ifeval::["{context}" == "osdk-working-bundle-images"]
 :golang:
 endif::[]
+ifeval::["{context}" == "osdk-java-tutorial"]
```

Comment:
I am not sure, how olm part is added to the document. Is it possible to add below content for running operator using OLM

```
$ export USERNAME=<container-registry-username>
$ export VERSION=0.0.1
$ export IMG=docker.io/$USERNAME/memcached-quarkus-operator:v$VERSION // location where your operator image is hosted
$ export BUNDLE_IMG=docker.io/$USERNAME/memcached-quarkus-operator-bundle:v$VERSION // location where your bundle will be hosted
```

You can see the tutorial here.

https://github.com/operator-framework/java-operator-plugins/blob/main/docs/tutorial.md#deploy-your-operator-with-olm

Reply:
I agree that the `IMG=` and `BUNDLE_IMG=` additions at the end of each command is a little excessive and verbose. I plan on re-working the "Running the Operator locally", "Running outside the cluster", and "Deploying on OLM" sections independently after the 4.11 release, simply because the files are shared between Go, Helm, Ansible, and Java SDK. I have created a task on the [OSDOCS board](https://issues.redhat.com/browse/OSDOCS-3847) for tracking this.

Reply:
That will be great! thank you for clarification.